browser/backend/README.md

```diff
@@ -15,7 +15,7 @@ Filename                  | Purpose                           | Information link
 
```

Comment:
In this file, there are a bunch of references to the "Data Browser." Would you consider updating it to "Data Portal" please?

Reply:
Our code dir structure uses `browser` instead of `portal`. Does the team prefer we also refactor this? @Bento007 @maniarathi .

Reply:
Regardless, that would be a larger refactor and not included in this PR. Resolving + following up offline